### PR TITLE
Contribution guide directory and folder naming

### PR DESCRIPTION
**Pull request by dplore**
Updated test folder and test file naming

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -95,9 +95,9 @@ Within each test directory, a README.md should document the test suite.  The
 `*.go` files should be named after an appropriate [github featureprofiles project](https://github.com/orgs/openconfig/projects/2/views/1?filterQuery=)
 item. For example:
 
-* `feature/interfaces/tests/rt_5_1_singleton_test/README.md` - documentation for the test suite
-* `tests/interfaces/tests/rt_5_1_singleton_test/singleton_test.go` implements issue **RT-5.1 Singleton Interface**.
-* `tests/interfaces/tests/rt_5_2_aggregate_test/aggregate_test.go` implements issue **RT-5.2 Aggregate Interface**.
+* `feature/interfaces/tests/singleton_test/README.md` - documentation for the test suite
```
Comment (liulk) [inline CONTRIBUTING.md:98]: feature/interfaces -> feature/interface

It looks like "network-instance" and "staticroute" are both singular, so interface should be singular also. The only outlier is "localaggregates" which probably should be "localaggregate"?

Comment (liulk) [inline CONTRIBUTING.md:98]: Also, my bad. I forgot we're using README.md for the test plan, so I've been using `foo_test.md` for `foo_test.go`. Do you feel strongly about using README.md, or should we just go along with `foo_test.md`?

Comment (dplore) [reply CONTRIBUTING.md:98]: I am ok with <testname>_test.md

**Comment by dplore** [comment]
If you approve Likai, feel free to merge

**Review by liulk — APPROVED** [review]

**Pull request merged**